zerver/views/user_settings.py

```diff
@@ -244,7 +244,7 @@ def set_avatar_backend(request: HttpRequest, user_profile: UserProfile) -> HttpR
 
 def delete_avatar_backend(request: HttpRequest, user_profile: UserProfile) -> HttpResponse:
     if avatar_changes_disabled(user_profile.realm) and not user_profile.is_realm_admin:
-        return json_error(AVATAR_CHANGES_DISABLED_ERROR)
+        return json_error(AVATAR_CHANGES_DISABLED_ERROR)  # discuss sgrep finding
```

Comment:
These could likely better be written as `raise AvatarChangesDisabledError()`, making a simple `JsonableError` subclass for this common string, if we wanted to remove the exclusion.